mcs/class/corlib/System.Runtime.InteropServices.RuntimeInformation/RuntimeInformation.cs

```diff
@@ -66,6 +66,11 @@ static RuntimeInformation ()
 				_osArchitecture = Environment.Is64BitOperatingSystem ? Architecture.X64 : Architecture.X86;
 				_processArchitecture = Architecture.X64;
 				break;
+			case "wasm":
```

Comment:
This looks good.  The only thing I would suggest here is to add a comment of why the `#if false` is there.